shm/Player.hpp

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include <algorithm>
+#include <memory>
+#include <numeric>
```

Comment:
This and `#include <algorithm>` should be imported in cpp file. It's unnecessary here

Reply:
Yeah, I tested my cod in one hpp file and forgot detete them. I will fix it. Thx!